src/extract-data-from-model.js

```diff
@@ -126,9 +125,6 @@ export default function extractDataFromModel(
           // the thunk action is created/dispatched
           def.thunkHandler = createThunkHandler(def, _r, injections, _aC);
 
-          // Register the thunk handler
-          set(path, actionThunks, def.thunkHandler);
-
```

Comment:
Maybe we don't have tests covering this? IDK what `actionThunks` are

Reply:
I've tested this branch against our codebase, and we get 0 failures - so I guess this is fine.

Reply:
Yeah, this is related to this change: https://github.com/ctrlplusb/easy-peasy/pull/876#discussion_r1358580832. The only place `actionThunks` was ever being used was here, the `thunkHandler` was being set on the object but then the object never got passed along to anything else. And if all the tests still pass, I'm thinking it's fine. Ctrl + Shift + F also turned up no other references to `actionThunks`. Obviously this isn't definitive because it could be passed to a func that calls it something else, but if deleting this line doesn't break anything else I think we're good.